src/dynamic_honey_badger/mod.rs

```diff
@@ -245,14 +249,20 @@ where
         self.verify_signature(sender_id, &sig, &kg_msg)?;
         let tx = SignedKeyGenMsg(self.start_epoch, sender_id.clone(), kg_msg, sig);
         self.key_gen_msg_buffer.push(tx);
-        self.process_output()
+        // FIXME: Remove the call to `process_output`. There wasn't any output from HB in this
+        // function.
+        self.process_output(Step::default())
```

Comment:
Feel free to do that right away. Or does something fail if you remove it?

Reply:
It did fail when I tried but that was some time ago. I can double-check.